Tests/run_tests.py

```diff
@@ -254,10 +256,13 @@ def main(argv):
             return 0
         if opt == "--offline":
             print("Skipping any tests requiring internet access")
-            # This is a bit of a hack...
-            import requires_internet
-            requires_internet.check.available = False
```

Comment:
I think leave this in place so that tests can check and abort immediately if we are running offline.